src/modules/auto-flickity.js

```diff
@@ -7,15 +7,10 @@
  *  |- FLICKITY CTN : .js-auto-flickity-slider-ctn
  *  |    |- CELL-CTN : .js-auto-flickity-ctn
  *  |    |    |- CELL (REPEATED): .js-auto-flickity-item
```

Comment:
The documentation you are referring to is for the old auto module for Flickity. I will commit another documentation for the JS component + Js module to use the new attribute system.